### DISCUSSION TITLE
Group and Task size and alignment.

### DISCUSSION THREAD

**Discussion by queezz** [csgoh/roadmapper#67] [Q&A]
Is there a way to modify vertical alignment of tasks? 
Now, as in Sample Roadmap example, the tasks are aligned to the top of a given group and there is an empty gap at the bottom of the group.

Is there a way to increase the group/task box? If the text is warped, it may get outside the colored box, which looks terrible.

**Comment by jenstreetman** [comment]
Along these lines, I'd love to limit the size of the group. It grows proportionately with the width of the roadmap, and for me that is undesirable.

**Comment by lozelin** [comment]
Anything new about this?
